kyuubi-server/src/main/scala/org/apache/kyuubi/session/KyuubiSessionManager.scala

```diff
@@ -314,7 +317,9 @@ class KyuubiSessionManager private (name: String) extends SessionManager(name) {
           metadata.className,
           metadata.requestArgs,
           Some(metadata),
-          fromRecovery = true)
+          fromRecovery = true,
+          // todo: subResources
+          subResources = Map.empty[String, String])
```

Comment:
seems all the uploaded sub resources are used by batch config,

can we add a reserved key into batch conf?

And get the uploaded files list(to cleanup) from batch conf when batch got recovered.

Reply:
then we do not need to persist the sub resources anymore.